chemprop/nn/metrics.py

```diff
@@ -122,8 +122,14 @@ def __call__(self, preds: Tensor, targets: Tensor, mask: Tensor, *args, **kwargs
 class AUROCMetric(Metric):
     minimize = False
 
+    def __init__(self, task: str) -> None:
+        self.task = task
+
     def __call__(self, preds: Tensor, targets: Tensor, mask: Tensor, *args, **kwargs):
-        return F.auroc(preds[mask], targets[mask].long())
+        return F.auroc(preds[mask], targets[mask].long(), task=self.task)
+    
+    def forward(self, preds, targets, mask) -> Tensor:
+        return self.__call__(preds, targets, mask)
```

Comment:
Do we have an idea where in the code this `forward()` method would be called? I have a feeling that because it's signature doesn't match the abstract one defined in `class Metric` then if part of the code does try to call this forward, there will be an error. Maybe we just don't implement a forward method.